### ISSUE TITLE
"Set selection anchor" with Voiceover

### ISSUE THREAD

**Issue by sana-ajani** [microsoft/vscode#99175]
Testing #98087

This is a nice feature, will definitely help with selecting large lines of text. When testing with voiceover, I noted two things. 

1) This is probably more of a macbook/touchbar thing, but I have to touch the escape key twice to cancel the anchor. 

2) Are you announcing that the anchor has been set? I'm not hearing that with voiceover currently.

**Comment by isidorn** [comment]
Thanks for testing and providing feedback

1. This is actually a vscode thing, since `ESC` is bound to quite a few actions each `ESC` does something else based on its priority. So for example if you have an anchor set and a find widget open, the first `ESC` will close the find widget the second will clear the anchor. I think this is fine in general, I did not want to give high priority to clearing the anchoring so we do not steal some keybinding
2. Yes we announce it, however VoiceOver sometimes does not read alerts, this is an issue on the VoiceOver side which I plan to investigate more into. For me it now reads nicely

Due to the above closing this issue.
